src/algorithm/mod.rs

```diff
@@ -82,11 +73,11 @@ impl Algorithm {
         if extract_five_aligned(player ^ &extract_captures(*enemy, *player, &self.patterns))
             .is_any()
             && extract_winning_move_capture(
-                *enemy,
-                *player,
-                node.get_opponent_captures(),
-                &self.patterns
-            )
+            *enemy,
+            *player,
+            node.get_opponent_captures(),
+            &self.patterns
+        )
```

Comment:
Those lines were indented the right way IMO since they are the parameters for the call to `extract_winning_move_capture()`